config/ember-try.js

```diff
@@ -2,83 +2,72 @@
 
 const getChannelURL = require('ember-source-channel-url');
 
-module.exports = function() {
-  return Promise.all([
-    getChannelURL('release'),
-    getChannelURL('beta'),
-    getChannelURL('canary')
-  ]).then((urls) => {
-    return {
-      useYarn: true,
```

Comment:
Looks like the Travis failures may be due to losing this line

Reply:
Yep, that definitely helped. Now it's just `v3.4` that is failing, saying it `{{exclaim-ui}}` isn't a helper 🤔 `v3.6` and up seem to work, but nothing below that